features/event-timing.yml

```
@@ -1,5 +1,5 @@
 name: Event timing
-description: The `event` and `first-input` performance entries and the `PerformanceEventTiming` API measures the latency of user input events, such as mouse clicks or keypresses, and the latency of the first user input. First Input Delay (FID) is a common metric for perceived responsiveness.
+description: The `event` and `first-input` performance entries of the `PerformanceEventTiming` interface measures the latency of user input events, such as mouse clicks or keypresses. Interaction to Next Paint (INP) is a common metric for perceived responsiveness calculated from the Event Timing API.
```

Some suggestions here to maintain consistency with other features.

Second, since the name of the metric changes, I think reworking the second sentence make the connection to the API itself becomes important and deserves emphasis. It's also consistent with entries like `largest-contentful-paint`.

```suggestion
description: The `event` and `first-input` performance entries and the `PerformanceEventTiming` API measures the latency of user input events, such as mouse clicks or keypresses. They're used to calculate Interaction to Next Paint (INP), a common metric for perceived responsiveness.
```